src/proposals/20250724/SparkEthereum_20250724.t.sol

```diff
@@ -0,0 +1,118 @@
+    address internal constant SPARK_USDS_VAULT                 = 0xe41a0583334f0dc4E023Acd0bFef3667F6FE0597;
+
+    constructor() {
+        id = "20250724";
+    }
+
+    function setUp() public {
+        setupDomains("2025-07-16T13:51:00Z");
+
+        deployPayloads();
+
+        // chainData[ChainIdUtils.Ethereum()].payload = 0x74e1ba852C864d689562b5977EedCB127fDE0C9F;
+    }
+
+    function test_ETHEREUM_WBTCChanges() public onChain(ChainIdUtils.Ethereum()) {
+        SparkLendContext memory ctx = _getSparkLendContext();
+
+        ReserveConfig[] memory allConfigsBefore = createConfigurationSnapshot('', ctx.pool);
+        ReserveConfig memory config = _findReserveConfigBySymbol(allConfigsBefore, 'WBTC');
+
+        assertEq(config.liquidationThreshold, 40_00);
+
+        executeAllPayloadsAndBridges();
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', ctx.pool);
+
+        config.liquidationThreshold = 35_00;
+
+        _validateReserveConfig(config, allConfigsAfter);
+    }
+
+    function test_BASE_morphoRateLimitChanges() public onChain(ChainIdUtils.Base()) {
+        bytes32 morphoUsdcVaultDepositKey = RateLimitHelpers.makeAssetKey(
+            ForeignController(Base.ALM_CONTROLLER).LIMIT_4626_DEPOSIT(),
+            Base.MORPHO_VAULT_SUSDC
+        );
+
+        _assertRateLimit(morphoUsdcVaultDepositKey, 50_000_000e6, 25_000_000e6 / uint256(1 days));
+
+        executeAllPayloadsAndBridges();
+
+        _assertRateLimit(morphoUsdcVaultDepositKey, 100_000_000e6, 50_000_000e6 / uint256(1 days));
+    }
+
+    function test_ETHEREUM_transferEthToMultisig() public onChain(ChainIdUtils.Ethereum()) {
+        uint256 balanceBefore = IERC20(Ethereum.WETH_ATOKEN).balanceOf(Ethereum.TREASURY);
+
+        assertGe(balanceBefore, 500e18);
+
+        executeAllPayloadsAndBridges();
+
+        assertEq(IERC20(Ethereum.WETH_ATOKEN).balanceOf(Ethereum.TREASURY),    0);
+        assertEq(IERC20(Ethereum.WETH_ATOKEN).balanceOf(LIQUIDATION_MULTISIG), balanceBefore);
+    }
+
+    function test_ETHEREUM_transferTokensToGrove() public onChain(ChainIdUtils.Ethereum()) {
+        uint256 buidlIbalanceBefore = IERC20(Ethereum.BUIDLI).balanceOf(Ethereum.ALM_PROXY);
+        uint256 jtrsyBalanceBefore  = IERC20(Ethereum.JTRSY).balanceOf(Ethereum.ALM_PROXY);
+
+        assertGe(buidlIbalanceBefore, 800_000_000e6);
+        assertGe(jtrsyBalanceBefore,  370_000_000e6);
+
+        executeAllPayloadsAndBridges();
+
+        assertEq(IERC20(Ethereum.BUIDLI).balanceOf(Ethereum.ALM_PROXY), 0);
+        assertEq(IERC20(Ethereum.JTRSY).balanceOf(Ethereum.ALM_PROXY),  0);
+        assertEq(IERC20(Ethereum.BUIDLI).balanceOf(GROVE_ALM_PROXY),    buidlIbalanceBefore);
+        assertEq(IERC20(Ethereum.JTRSY).balanceOf(GROVE_ALM_PROXY),     jtrsyBalanceBefore);
+    }
+
+    function test_ETHEREUM_morpho_PTSPKUSDS24SEP2025Onboarding() public onChain(ChainIdUtils.Ethereum()) {
```

Comment:
Why is this test in a different format than the others? The others seem to follow the format 

```
// assert state before
executeAllPayloadsAndBridges();
// assert state after
```

Whereas this one never calls `executeAllPayloadsAndBridges()`.

Reply:
Thanks, will review.